data.json

```
@@ -259,5 +259,14 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
+    "name": "opsdroid",
+    "difficulty": "hard",
```

```suggestion
    "difficulty": "medium",
```

Will reflect other difficulties (i.e. Umbraco)